CHANGELOG.md

```diff
@@ -59,6 +57,7 @@
 - `write_slice`, `read_slice`, `read_slice_1d`, `read_slice_2d` now take any object
   convertible to `Selection` (instead of `SliceInfo`).
 - `Dataset::chunks` has been renamed to `Dataset::chunk`
+- `hdf5-types` uses `const-generics` for arrays, allowing arbitrary sized arrays.
```

Comment:
Probably worth mentioning in the changelog that MSRV is now 1.51?

Reply:
Given that `const-generics` is no longer a crate feature, perhaps rewrite this as?

```markdown
- Const generics support (Rust 1.51): `hdf5-types` now uses const generics for array types, 
  allowing fixed-size arrays of arbitrary sizes.
```